packages/devextreme/js/__internal/ui/html_editor/modules/m_toolbar.ts

```diff
@@ -484,6 +487,9 @@ if (Quill) {
       const options = {
         dataSource,
         disabled: isMenuDisabled,
+        onSubmenuShown: ({ submenuContainer }: SubmenuShownEvent): void => {
+          $(submenuContainer).parent().addClass(TOOLBAR_AI_OVERLAY_CONTENT_CLASS);
+        },
```

Comment:
It's a crutch to me, isn't there another solution?
While UI issues is in Menu, shouldn't the solution be in Menu?